sites/docs/docs/components/custom-components.md

```diff
@@ -68,6 +68,49 @@ Add a folder called `components/` in the root of your project. This is where Evi
 <BarChart data="{query}" />
 ```
 
+## Modifying an existing component
+Adapting any existing components to your needs is straightforward as well. You can achieve this by duplicating an existing component into your `/components` directory, modifying that new component file, and importing the new component as needed in your project.
```

Comment:
~~Adapting any existing components to your needs is straightforward as well. You can achieve this by duplicating an existing component into your `/components` directory, modifying that new component file, and importing the new component as needed in your project.~~
Modify an existing component by copying the component's source code into the `/components` directory, modifying it, and importing the new component as needed in your project.